src/Domain/CoordinateMaps/AngularCompression.hpp

```diff
@@ -0,0 +1,92 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <cstddef>
+#include <limits>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Utilities/TypeTraits.hpp"
+
+namespace PUP {
+class er;
+}  // namespace PUP
+
+namespace CoordinateMaps {
+
+/*!
+ * \ingroup CoordinateMapsGroup
+ *
+ * \brief Redistributes gridpoints on the sphere.
+ * \image html AngularCompression.png "A sphere with an `aspect_ratio` of 3."
+ *
+ * \details A mapping from the sphere to itself which depends on a single
+ * parameter, the `aspect_ratio` \f$\alpha\f$. Gridpoints located at an angle
+ * \f$\theta\f$ from the pole are mapped to a new angle \f$\theta'\f$ satisfying
+ * \f$\textrm{tan}\theta' = \alpha \textrm{tan}\theta\f$. Note that the aspect
+ * ratio must be positive.
```

Comment:
I would find it helpful if the comment said which way the points move — towards or away from the equator — for aspect_ratio greater or less than one. Primarily, this would help give intuition as to whether the aspect ratio is height/width or width/height; I know this is mentioned in the test, but it would help to have it right here in the documentation.

(Opinion: I would put this before the note on aspect_ratio being positive.. both of these could be in a new paragraph?)